velox/tpch/gen/dbgen/include/tpch_constants.hpp

```diff
@@ -1,9 +1,29 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 /* THIS FILE WAS AUTOMATICALLY GENERATED BY generate_csv_header.py */
 
 #pragma once
 
 namespace facebook::velox::tpch::dbgen {
 
+#define TPCH_QUERY(id) reinterpret_cast<const char*>(TPCH_QUERIES_##id)
+#define TPCH_ANSWER_SF1(id) reinterpret_cast<const char*>(TPCH_ANSWERS_SF1_##id)
+#define TPCH_ANSWER_SF0_1(id) reinterpret_cast<const char*>(TPCH_ANSWERS_SF0_1_##id)
+#define TPCH_ANSWER_SF0_01(id) reinterpret_cast<const char*>(TPCH_ANSWERS_SF0_01_##id)
```

Comment:
@aditi-pandit i noticed you removed the undefs. Let's keep them to avoid leaking the pragmas.

Reply:
Moved the defines and undefs closer to use, so that the scope is restricted to the minimum.